domains/lumi.json

```diff
@@ -7,5 +7,6 @@
     },
     "record": {
         "CNAME": "lumidevelopment.github.io"
+        "TXT": "vc-domain-verify=lumi.is-a.dev,38de86570893517719b8"
```

Comment:
Can't use CNAME and TXT together.
Please read is-a.dev docs on how to add the Vercel TXT